src/main/java/com/google/finapp/Args.java

```diff
@@ -0,0 +1,31 @@
+package com.google.finapp;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+
+@Parameters(separators = "=")
+class Args {
+
+  @Parameter(names = {"--port", "-p"})
+  int port = 8080;
```

Comment:
Perhaps this is too much boilerplate, but I'd expect thee to be private with getters in Java. Thoughts?

Reply:
I've moved this to the main class so that we don't need explicit getters as such.